admin/jqadm/src/Admin/JQAdm/Product/Image/Standard.php

```diff
@@ -479,13 +479,19 @@ protected function fromArray( \Aimeos\MShop\Product\Item\Iface $product, array $
 		{
 			if( !isset( $listItems[$listid] ) )
 			{
+
+				// There are new media file at the form. Manage it
+				// Let's create a new one media item
 				$litem = clone $listItem;
+				$item = clone $mediaItem;
 
 				if( ( $refId = $this->getValue( $data, 'product.lists.refid/' . $idx ) ) !== null ) {
-					$item = $mediaManager->getItem( $refId, ['attribute'] ); // copy existing item
-				} else {
-					$item = clone $mediaItem;
+				    // There are existing $refId
```

Comment:
Can you please replace the four spaces by a tab to make the coding style check happy?

Reply:
Hm... I thought i did it already. You don't see it now, yes?

Reply:
I did request with four spaces, then I saw you are use tab instead of 4 spaces. I fixed it and submit code second time.

Reply:
No, still there just before the first non-space characters. You can also check the tests (at the bottom): https://travis-ci.org/aimeos/ai-admin-jqadm/jobs/331495908

Reply:
I'm really really sorry, it's my first experience

Reply:
Please check it out. I hope it's okay now. I think i have to configure my IDE to the project code style